diff --git a/node_modules/@types/react-beautiful-dnd/index.d.ts b/node_modules/@types/react-beautiful-dnd/index.d.ts
index 753c10f..fbbe482 100755
--- a/node_modules/@types/react-beautiful-dnd/index.d.ts
+++ b/node_modules/@types/react-beautiful-dnd/index.d.ts
@@ -744,3 +744,8 @@ export function resetServerContext(): void;
 export const useMouseSensor: Sensor;
 export const useTouchSensor: Sensor;
 export const useKeyboardSensor: Sensor;
+
+export const DOM: {
+    setHead: (head: HTMLElement | ShadowRoot) => void;
+    setBody: (body: HTMLElement | ShadowRoot) => void;
+};
